admin/src/components/Tables/SearchUserTable.vue

```diff
@@ -48,20 +48,20 @@
       </template>
 
       <template #row-details="row">
-        <b-card ref="rowDetails" class="shadow-lg pl-3 pr-3 mb-5 bg-white rounded">
-          <b-tabs content-class="mt-3">
-            <b-tab :title="$t('creation')" active :disabled="row.item.deletedAt !== null">
-              <creation-formular
+        <BCard ref="rowDetails" class="shadow-lg pl-3 pr-3 mb-5 bg-white rounded">
+          <BTabs content-class="mt-3">
+            <BTab :title="$t('creation')" active :disabled="row.item.deletedAt !== null">
+              <CreationFormular
```

Comment:
Since these are our regular components from the project maybe it could stay in kebab case to distinguish it from bootstrap components? WDYT?